neo/IO/Caching/DataCache.cs

```diff
@@ -62,7 +63,8 @@ public void Add(TKey key, TValue value)
 
         public void Commit()
         {
-            foreach (Trackable trackable in GetChangeSet())
+            Parallel.ForEach(GetChangeSet(), (trackable) =>
```

Comment:
I don't think it works. We use batch in inner db.

Reply:
Maybe i can remove this, and only maintain the `Contract Update` changes?